WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/AttributedText.swift

```diff
@@ -0,0 +1,150 @@
+import SwiftUI
+
+/*
```

Comment:
I think we have an "Open Source licenses" section in Settings. Should this be in there?

Reply:
Good call, I didn't know about that. I've added a line in 80710432e